src/components/ui/MimEstimatePrice.vue

```diff
@@ -180,7 +179,7 @@ export default {
         this.parsedAmount.toString()
       );
 
-      this.itsProfit = this.itsClose ? price >= 1 : price <= 1;
+      this.itsProfit = this.itsClose ? price <= 1 : price >= 1;
```

Comment:
This should be `this.itsProfits = price > 1`

Reply:
updated